src/dotnet/commands/dotnet-restore3/Program.cs

```diff
@@ -97,6 +103,13 @@ public static int Run(string[] args)
                     msbuildArgs.Add($"/p:RestoreIgnoreFailedSources=true");
                 }
 
+                // By default restore is recursive, if set to false turn it off.
```

Comment:
Let's invert the bool so the default is false. It makes more sense to have the default value of Boolean options be false.